source/libraries/ContractInterfaces.ts

```diff
@@ -6561,4 +6561,3 @@
     export function ContractFactory(connector: Connector, accountManager: AccountManager, address: string, defaultGasPrice: BN): Controlled {
         return new Controlled(connector, accountManager, address, defaultGasPrice);
     }
-    
```

Comment:
Nit: should probably keep extra line at the end of file since the linter expects this.